v3/src/components/app.tsx

```diff
@@ -1,21 +1,22 @@
 import React from "react"
 import { CaseTable } from "./case-table/case-table"
-import { Container } from "./container"
-import { DataSummary } from "./data-summary"
-import { gDataBroker } from "../data-model/data-broker"
-import { DataSet, toCanonical } from "../data-model/data-set"
-import { Text } from "./text"
-import { useDropHandler } from "../hooks/use-drop-handler"
-import { useSampleText } from "../hooks/use-sample-text"
+import {Container} from "./container"
+import {DataSummary} from "./data-summary"
+import {gDataBroker} from "../data-model/data-broker"
+import {DataSet, toCanonical} from "../data-model/data-set"
+import {Graph} from "./graph/graph"
+import {Text} from "./text"
+import {useDropHandler} from "../hooks/use-drop-handler"
+import {useSampleText} from "../hooks/use-sample-text"
```

Comment:
I'm guessing that the IntelliJ auto-formatter removed the spaces before/after the braces here. I'd recommend turning that off, as I don't think that's something we want to mandate.